R/openapi-types.R

```diff
@@ -0,0 +1,132 @@
+# calculate all OpenAPI type information at once and use created information throughout package
+dataTypesInfo <- list()
+dataTypesMap <- list()
```

Comment:
May we use a more specific name? Maybe `apiTypesInfo`? (`sub("data", "api", .)`)

It's a lot of name changes, but it helps when reading to supply context

Reply:
I was in the process a decoupling the mountui part and de renaming, reorganizing part.

Also wondering what would be a good name for dataType.

It refers to OpenAPI data types. If you think apiTypes would work, I'm down for it.

Don't worry about replacing stuff. Naming things is the hardest part.

Reply:
> Also wondering what would be a good name for dataType.

`apiType` helps a lot. It lets me know that it's prolly a string value describing the OpenAPI Spec type.

`dataType` feels like an output of `typeof()` or something unrelated to OpenAPI

> I was in the process a decoupling the mountui part and de renaming, reorganizing part.

Great! Feel free to make those changes there. 😄

Reply:
I am hoping to refer to all things `OpenAPI` by using `api` in the variable name.

(Things with `plumber` are `pr` or `router`.)